schema/deploy/shipping/views.sql

```diff
@@ -108,11 +108,13 @@ create or replace view shipping.presence_absence_result_v1 as
 
     select sample.identifier as sample,
            target.identifier as target,
-           present
+           present,
+           organism.lineage
 
     from warehouse.sample
     join warehouse.presence_absence using (sample_id)
     join warehouse.target using (target_id)
+    join warehouse.organism using (organism_id)
```

Comment:
This needs to be a left join so we don't exclude results for targets that aren't yet linked to an organism.

Reply:
Done.